### PR TITLE
Updated port mirror service.

### PR DESCRIPTION
**Pull request by zlion**
Updated the network handler to support one-to-one port mirroring including VLANs.
Need more underlying code updates to support multiple-to-one port mirroring.

### CODE REVIEW

File: fabric_am/test/test_network_handler.py [hunk 1]
```diff
@@ -1060,14 +1060,15 @@ def test_PortMirror(self):
...
         sliver.set_layer(NSLayer.L2)
 
         # mirror_port is the name of the port being mirrored - actual name the way
         # service definition needs it. It comes directly from ASM network service sliver
         # whatever the right name is - user must to know it when creating a slice
-        sliver.mirror_port = "TwentyFiveGigE0/0/0/24"
+        #sliver.mirror_port = "HundredGigE0/0/0/17"
+        sliver.mirror_port = "HundredGigE0/0/0/17.100"
         # direction also comes from ASM network service sliver
         sliver.mirror_direction = MirrorDirection.Both
```
Comment (kthare10) [inline fabric_am/test/test_network_handler.py:1074]: `sliver.mirror_port` would still be `HundredGigE0/0/0/17`
`sliver.mirror_vlan` would now contain  vlan tag `100` which should be used.

File: fabric_am/handlers/net_handler.py [hunk 2]
```diff
@@ -904,8 +904,16 @@ def __portmirror_create_data(self, sliver: NetworkServiceSliver, service_name: s
...
                 "direction": direction}
+        interface = self.parse_interface_name(sliver.mirror_port)
+        if "outervlan" in interface or "innervlan" in interface:
+            if 'from-interface-vlan' not in data:
+                data['from-interface-vlan'] = []
+            data['from-interface-vlan'].append(self.parse_interface_name(sliver.mirror_port))
+        else:
+            if 'from-interface' not in data:
+                data['from-interface'] = []
+            data['from-interface'].append(self.parse_interface_name(sliver.mirror_port))
```
Comment (kthare10) [inline fabric_am/handlers/net_handler.py:916]: Please use `sliver.mirror_vlan` for vlan tag.

File: fabric_am/test/test_network_handler.py (hunk 1 above)
Comment (zlion) [reply fabric_am/test/test_network_handler.py:1075]: Changed to sliver.mirror_vlan.

File: fabric_am/handlers/net_handler.py (hunk 2 above)
Comment (zlion) [reply fabric_am/handlers/net_handler.py]: Changed.

**Review by kthare10 — APPROVED** [review]
Looks good! Tested on Dev!

**Pull request merged**